types/arangodb/index.d.ts

```diff
@@ -659,7 +659,7 @@ declare namespace ArangoDB {
         document(
             selectors: ReadonlyArray<string | DocumentLike>
         ): Array<Document<T>>;
-        exists(name: string): boolean;
+        exists(name: string): Document<T> | boolean;
```

Comment:
This should be `DocumentMetadata | false` then.

Reply:
Thanks, fixed!